<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsMenu</class>
 <widget class="QMainWindow" name="StatisticsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resouces.qrc">:/assets/hamburger_logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heaven</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>-20</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Food</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>451</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cea mai profitabilă zi este:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Produsul cel mai bine vândut este :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Statistici totale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_most_profitable_day">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_most_sold_prod">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Media vâznări produselor :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_avergare_sales">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView::item:selected {
      border: 1px solid #6a6ea9;
  }

  QListView::item:selected:!active {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #ABAFE5, stop: 1 #8588B2);
  }

  QListView::item:selected:active {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #6a6ea9, stop: 1 #888dd9);
  }

  QListView::item:hover {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #FAFBFE, stop: 1 #DCDEF1);
  }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show_daily_stats">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#5F5F5F;
border: 2px solid gray;
border-radius:12px;
padding:10 10 10 8px;

}
QPushButton::pressed{
border: 3px solid black;
}
QPushButton::hover{
 background-color: #B8B8B8;
 color: 	black;
}</string>
    </property>
    <property name="text">
     <string>Arată Statisticile zilnice</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total vânzări Înregistrate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total profit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_most_profitable_day_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>420</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_most_profitable_day_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_daily_stats">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>831</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Produse vândute</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_sold_products">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>251</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView::item:selected {
      border: 1px solid #6a6ea9;
  }

  QListView::item:selected:!active {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #ABAFE5, stop: 1 #8588B2);
  }

  QListView::item:selected:active {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #6a6ea9, stop: 1 #888dd9);
  }

  QListView::item:hover {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #FAFBFE, stop: 1 #DCDEF1);
  }</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_statistic_date">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>----</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Statistici in data de:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_lei2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>470</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>lei</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lei1">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>420</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>lei</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resouces.qrc"/>
 </resources>
 <connections/>
</ui>
